.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "secrets_secure"]
 	path = secrets_secure
 	url = git@github.com:protocol/infrastructure-secrets.git
+[submodule "ansible/roles/geerlingguy.docker"]
+	path = ansible/roles/geerlingguy.docker
+	url = git@github.com:geerlingguy/ansible-role-docker.git
```

Comment:
I disagree, think it's actually better practice to rely on code that's been tested by other people than to write everything.

Reply:
We should at least fork them so that we can control when they are updated and can vet those updates.

Reply:
> We should at least fork them so that we can control when they are updated and can vet those updates.

The module is currently added as a GIT submodule, is that good enough for now ? We can vet new releases and only update the submodule hash if we're satisfied.